DB/Schema.lua

```diff
@@ -28,8 +28,53 @@ function DatabaseSchema:IsValidItem(entry)
 		return false
 	end
 
-	if not Item:HasZone(entry) then
-		Rarity:Print("Zone is missing")
+	if Item:IsCollectionItem(entry) and not Item:IsValidCollectionItem(entry) then
+		Rarity:Print("Collection settings are inconsistent")
+		return false
+	end
+
+	if Item:IsArchaeologyItem(entry) and not Item:IsValidArchaeologyItem(entry) then
+		Rarity:Print("Archaeology settings are inconsistent")
+		return false
+	end
+
+	if Item:HasUseMethod(entry) and not Item:IsValidUseMethodItem(entry) then
+		Rarity:Print("Found item with use method, but it's items are not valid.")
+		return false
+	end
+
+	if Item:IsCovenantSpecificItem(entry) and not Item:IsValidCovenantItem(entry) then
+		Rarity:Print("Covenant settings are inconsistent")
+		return false
+	end
+
+	if Item:IsZoneItem(entry) and not Item:IsValidZoneItem(entry) then
+		Rarity:Print("Zone settings are inconsistent")
```

Comment:
See above ("inconsistent" too vague?)

Reply:
Rephrased this in 7bb53d56902e48e69116e48dbe8cca91c4fe8179.